modules/compatibility-tag/views/plugin-update-message-compatibility.php

```diff
@@ -0,0 +1,47 @@
+<?php
+use Elementor\Core\Utils\Version;
+use Elementor\Modules\ExtensionsCompatibility\Module;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * those variables was declared in 'in_plugin_update_message' method that includes the current view file.
+ *
+ * @var Version $new_version
+ * @var array $untested_extensions
+ * @var string $header
+ */
+?>
+<hr class="e-major-update-warning__separator" />
+<div class="e-major-update-warning">
+	<div class="e-major-update-warning__message">
+		<strong>
+			<?php esc_html_e( 'Compatibility Alert', 'elementor' ); ?>
+		</strong> -
+		<?php
+		/* translators: %s: Elementor version */
+		echo sprintf(
+			__( 'some of the plugins you’re using have not been tested with the latest version of Elementor (%s). To avoid issues, make sure they are all up to date and compatible before updating Elementor.', 'elementor' ),
```

Comment:
```suggestion
			__( 'Some of the plugins you’re using have not been tested with the latest version of Elementor (%s). To avoid issues, make sure they are all up to date and compatible before updating Elementor.', 'elementor' ),
```